assets/js/components/settings/settings-admin.js

```diff
@@ -19,20 +19,68 @@
 /**
  * WordPress dependencies
  */
-import { Fragment } from '@wordpress/element';
+import { Fragment, useCallback } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
 
 /**
  * Internal dependencies
  */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as CORE_USER } from '../../googlesitekit/datastore/user/constants';
 import Layout from '../layout/layout';
+import { Cell } from '../../material-components';
 import OptIn from '../optin';
 import VisuallyHidden from '../VisuallyHidden';
 import ResetButton from '../ResetButton';
+import UserInputPreview from '../user-input/UserInputPreview';
+import {
+	USER_INPUT_QUESTION_GOALS,
+	USER_INPUT_QUESTION_HELP_NEEDED,
+	USER_INPUT_QUESTION_POST_FREQUENCY,
+	USER_INPUT_QUESTION_ROLE,
+	USER_INPUT_QUESTION_SEARCH_TERMS,
+} from '../user-input/util/constants';
+import { STORE_NAME as CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import UserInputSettings from '../notifications/UserInputSettings';
+const { useSelect } = Data;
+
+const questions = [
+	USER_INPUT_QUESTION_ROLE,
+	USER_INPUT_QUESTION_POST_FREQUENCY,
+	USER_INPUT_QUESTION_GOALS,
+	USER_INPUT_QUESTION_HELP_NEEDED,
+	USER_INPUT_QUESTION_SEARCH_TERMS,
+];
```

Comment:
This array is duplicated with the same array in `UserInputQuestionnaire` https://github.com/google/site-kit-wp/blob/878a304b92fd13a642894c6938657e1e1932c5f9/assets/js/components/user-input/UserInputQuestionnaire.js#L47-L53

This risks these two being out of sync if the array changes in one place and not the other.

Let's extract this array into the user input constants and reference that in both places rather than duplicating it in both.